src/zkevm_specs/tx.py

```diff
@@ -137,30 +158,33 @@ def assign(cls, signature: KeyAPI.Signature, pub_key: KeyAPI.PublicKey, msg_hash
         self_msg_hash = Secp256k1ScalarField(int.from_bytes(msg_hash, "big"))
         return cls(self_signature, self_pub_key, self_msg_hash)
 
-    def verify(self, is_enabled: FQ, assert_msg: str):
-        msg_hash = bytes(reversed(self.msg_hash.limbs))
-        sig_r = int.from_bytes(self.signature[0].limbs, "little")
-        sig_s = int.from_bytes(self.signature[1].limbs, "little")
+    def verify(self, assert_msg: str):
+        msg_hash = bytes(reversed(self.msg_hash.to_le_bytes()))
+        sig_r = int.from_bytes(self.signature[0].to_le_bytes(), "little")
+        sig_s = int.from_bytes(self.signature[1].to_le_bytes(), "little")
         signature = KeyAPI.Signature(vrs=[0, sig_r, sig_s])
```

Comment:
`v` always zero?

Reply:
Here this signature object is used to verify a signature against the message hash and public key.  For an ECDSA signature verification, the `v` value is not needed (Actually an ECDSA signature is only [`(r, s)`](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Signature_generation_algorithm)).  That's why I set `v = 0`, because it's not used here.
The `v` value is used when recovering the public key from a signature and message.